src/Foundation/NSMetadataItem.cs

```diff
@@ -242,7 +262,13 @@ bool GetBool (NSString key)
 #if NET
 		public bool? UbiquitousItemDownloadRequested {
 #else
-		[iOS (8,0)][Mac (10,10)]
+#if NET
+		[SupportedOSPlatform ("ios8.0")]
```

Comment:
This is inside a !NET block from the #if NET just above, so it won't do the right thing. These attributes should be moved into the block above instead.